hypha/apply/users/utils.py

```diff
@@ -1,5 +1,6 @@
 import string
 
+import nh3
```

Comment:
My IDE complains that we do not have `nh3` in the project requirements. We have `django-nh3`. Are we importing a transitive dep here?

Reply:
Added in 3ef7a738daee00c0555c280734a09a1139de8295